### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-03-24

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#74] [General]
# OMI glTF Working Group Meeting 3/24/2022

This meeting will be on 3/24/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- OMI_audio_emitter
  - Khronos meeting debrief
  - KHR_audio proposal https://github.com/omigroup/gltf-extensions/pull/73
- Text Extension
  - https://github.com/omigroup/gltf-extensions/issues/62
- Collider extension check-in
  - https://github.com/omigroup/gltf-extensions/pull/63
- Avatar Animation Retargeting
  - https://github.com/omigroup/gltf-extensions/issues/44
- Visibility extension proposal
  - https://github.com/omigroup/gltf-extensions/discussions/72#discussioncomment-2383988

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by KooIaIa** [comment]
extension sub-groups / sub-sets
